### PR TITLE
Update search documentation for course authors and learners

### PR DESCRIPTION
**Pull request by srpearce**
Adding cross-course search information.

### CODE REVIEW

File: en_us/shared/students/SFD_course_search.rst [hunk 1]
```diff
@@ -4,24 +4,53 @@
...
 #####################
 
-Much of the content of your edX course is searchable, including course text and
-video transcripts. To search your edX course content, go to the **Courseware**
-tab, and then enter a search term in the **Search** field in the upper-left
-corner of the page. Your search term can contain more than one word.
+You can search much of the content of your edX courses, including the course
+title, description, text, and video transcripts. You can search for a term in
+an individual course, or you can search for that term in all of the courses
+that you are enrolled in, whether the course is active or archived. When the
```
Comment (srpearce) [inline en_us/shared/students/SFD_course_search.rst:10]: @dino-cikatic Is it true that archived courses will be searched as well?

**Comment by srpearce** [comment]
@dino-cikatic Please review the text in this PR and let me know if it's technically accurate. Also, please let me know if there's a sandbox where I can see this feature. Thanks!

File: en_us/shared/students/SFD_course_search.rst (hunk 1 above)
Comment (dino-cikatic) [inline en_us/shared/students/SFD_course_search.rst:10]: Yes, archived courses one was enrolled in will also be searched.

File: en_us/shared/students/SFD_course_search.rst [hunk 2]
```diff
@@ -4,24 +4,63 @@
...
-After you enter your search term, your search results appear in the right
-pane. The search results include your search term, a brief excerpt from the
-surrounding text, and the place in the course where the search term appears. To
-view your search term in the body of the course, click **View** to the right of
-the search result.
+.. note:: Course exercises and problems, including homework and exam problems, 
+ are not currently searchable.
 
 Many searches return your search term and similar terms. For example, a search
 for "grade" returns results for "grades" and "graded".
```
Comment (dino-cikatic) [inline en_us/shared/students/SFD_course_search.rst]: Might be a good idea to indicate here it's not just similar, it returns results containing/including search term string. (so it won't return grad/rade..)

File: en_us/shared/building_and_running_chapters/building_course/search_course.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+course title, description, course text, and video transcripts. Learners can
+search inside individual courses, or they can search all the courses that they
+are enrolled in at one time. When learners search all their courses at one
+time, they can filter search results based on course subject, type, and
+language.
+
+Before learners can search your course, Studio must index your course content.
+Studio indexes your course content automatically every time you make a change
+to the content. You can also manually begin an indexing operation in Studio.
+To do this, select **Index** in the upper right corner of the **Course
```
Comment (dino-cikatic) [inline en_us/shared/building_and_running_chapters/building_course/search_course.rst]: I'm not sure, but I believe this used to be Reindex. Any chance you can double check this with translations team?

**Pull request merged**